packages/orchestration/src/types.ts

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable no-use-before-define -- fine for types */
+// Ambients
+import '@agoric/zoe/exported.js';
+
+import type { Amount, Brand } from '@agoric/ertp/exported.js';
+import type {
+  CoinAmount,
+  CosmosChainInfo,
+  CosmosStakingFacet,
+  Denom,
+  IcaAccount,
+} from './cosmos-api.js';
+import type { EthChainInfo } from './ethereum-api.js';
+import type { ICQConnection } from './exos/icqConnectionKit.js';
+import type {
+  ChainAddress,
+  OrchestrationAccountI,
+  TransferMsg,
+} from './orchestration-api.js';
+
+export type * from './cosmos-api.js';
+export type * from './exos/chainAccountKit.js';
+export type * from './exos/icqConnectionKit.js';
+export type * from './orchestration-api.js';
```

Comment:
this looks circular. I can imaging that's not a ts error, but it seems unhelpful for thinking about things.

Reply:
Is this a blocking request for the PR? I can untangle the circularity but I wouldn't want to do that until the API surface is agreed upon

Reply:
> Is this a blocking request for the PR?

no.

I'm trying to understand the proposal.